src/main/scala/com/cognite/sdk/scala/v1/sessions.scala

```diff
@@ -34,13 +34,14 @@ final case class BindSessionRequest(
 )
 
 final case class RefreshSessionRequest(
+    sessionId: Long,
     sessionKey: String
 )
 
 final case class SessionTokenResponse(
     id: Long,
     accessToken: String,
     expiresIn: Long,
-    mustRefreshIn: Long,
+    mustRefreshIn: Option[Long],
```

Comment:
Sure this is optional?

Reply:
Yes this field is not returned yet, its implementation is on-going https://github.com/cognitedata/api-auth/pull/1624